### PR TITLE
added actions to furnishings controller

### PR DESCRIPTION
**Pull request by hebazmalik**

### CODE REVIEW

File: app/controllers/furnishings_controller.rb [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+  def destroy
+    @furnishing = Furnishing.find(params[:id])
+    @furnishing.destroy
+    redirect_to furnishings_path
+  end
+
+  private
+
+  def strong_params
+    params.require(:furnishing).permit(:title, :description, :price, :location, :furnishing_type)
```
Comment (tim94white) [inline app/controllers/furnishings_controller.rb:44]: need to end strong params def

**Review by tim94white — COMMENTED** [review]
one comment line 44

File: app/controllers/furnishings_controller.rb (hunk 1 above)
Comment (hebazmalik) [reply app/controllers/furnishings_controller.rb:44]: added

**Review by tim94white — COMMENTED** [review]
nice

File: app/controllers/furnishings_controller.rb [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+    @furnishing = Furnishing.find(params[:id])
+  end
+
+  def new
+    @furnishing = Furnishing.new
+  end
+
+  def create
+    @furnishing = Furnishing.new(strong_params)
+    if @furnishing.save
```
Comment (Leooogo) [inline app/controllers/furnishings_controller.rb:16]: after line 15, don't we need to add something related to user cause furnishing belongs to user.

File: app/controllers/furnishings_controller.rb [hunk 3]
```diff
@@ -1,2 +1,45 @@
...
+      render :new
+    end
+  end
+
+  def edit
+    @furnishing = Furnishing.find(params[:id])
+  end
+
+  def update
+    @furnishing = Furnishing.find(params[:id])
```
Comment (Leooogo) [inline app/controllers/furnishings_controller.rb:28]: line 28 and 29, the parameter doesn't have to be the same thing? I'm not sure, just I'm asking.

File: app/controllers/furnishings_controller.rb (hunk 1 above)
Comment (Leooogo) [inline app/controllers/furnishings_controller.rb:44]: do we have 'location' in furnishing, cause I've seen only title, description, price and furnishing_type.

**Review by Leooogo — COMMENTED** [review]
please check!

File: app/controllers/furnishings_controller.rb (hunk 1 above)
Comment (hebazmalik) [reply app/controllers/furnishings_controller.rb:44]: just checked the schema.rb and location is one of the attributes for furnishings

**Pull request merged**